numba/targets/arrayobj.py

```diff
@@ -904,6 +904,75 @@ def array_argmax_impl(arry):
     return context.compile_internal(builder, array_argmax_impl, sig, args)
 
 
+@builtin
+@implement(numpy.median, types.Kind(types.Array))
+def array_median(context, builder, sig, args):
+    from numba import typing, int64, float64
```

Comment:
Time for cosmetics :) int64 and float64 are already available as `types.int64` and `types.float64` (also, note that we tend to use `types.intp` for indices, rather than int64).